duckduckgo/Android
# Added LE certificates and perform chain validation

Task/Issue URL: https://app.asana.com/0/0/1199564873039374/f
Tech Design URL: 
CC: 

**Description**:
This PR:

* Bundled LE root and intermediate certificates with the APK

* Added LE root certificates to the `network_security_config.xml`. This covers API >= 24

* Added manual chain verification for `SSL_UNTRUSTED` errors that covers API < 24

**Steps to test this PR**:
API <= 25
1. Install app from develop branch
2. Navigate to https://valid-isrgrootx1.letsencrypt.org/
3. Verify that page does not load

API > 25
1. Install/upgrade app from this PR branch
2. Perform sanity checks navigating to your usual sites
3. Navigate to https://valid-isrgrootx1.letsencrypt.org/
4. Verify page loads normally

API <= 25
1. Install/upgrade app from this PR branch
2. Perform sanity checks navigating to your usual
3. Navigate to https://valid-isrgrootx1.letsencrypt.org/
4. Verify page loads normally

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
